### ISSUE TITLE
[files app] a minor cosmetic "naming"-bug

### ISSUE THREAD

**Issue by DJCrashdummy** [nextcloud/server#6307]
### Steps to reproduce
1. open the files-app in the web-interface at nextcloud 12 (nextcloud 11 is NOT affected!)
2. choose an other group (like shared, favorites, etc. or even deleted) and the title of the site/tab gets accordingly renamed.
3. now choose "all files" within the side-menu of the app, not the general app-button and the site-/tab-name stays the same as before.

### Expected behaviour
the site/tab should be renamed "files" like it is when you access the files-app.

### Actual behaviour
see _Steps to reproduce 3._

### Server configuration

**Operating system**: Linux 3.13.0-128-generic Ubuntu (i guess my hoster told me some time ago it is Ubuntu 14.04)

**Web server:** Apache

**Database:** mysql 5.6.33

**PHP version:** 7.0.20

**Nextcloud version:** 12.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** updated just from 12.0.1

**Where did you install Nextcloud from:** hoster-script

**Signing status:**
```
No errors have been found.
```

**List of activated apps:**
<details>
<summary>App list</summary>

```
activity
bruteforcesettings
calendar
comments
contacts
dav
federatedfilesharing
federation
files
files_pdfviewer
files_sharing
files_texteditor
files_trashbin
files_versions
files_videoplayer
firstrunwizard
gallery
logreader
lookup_server_connector
nextcloud_announcements
notifications
oauth2
password_policy
provisioning_api
serverinfo
sharebymail
twofactor_backupcodes
updatenotification
user_external
workflowengine

apporder
checksum
files_downloadactivity
files_opds
files_reader
keeweb
metadata
notes
qownnotesapi
quota_warning
socialsharing_email
spreed
tasks
telephoneprovider

**INACTIVE:**
admin_audit
systemtags
encryption
files_external
user_ldap
theming
survey_client

ransomware_protection
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
<?php
$CONFIG = array (
  'instanceid' => 'xxxxxxxxxxxx',
  'passwordsalt' => 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
  'secret' => 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
  'trusted_domains' => 
  array (
    0 => 'cloud.mydomain.tld',
  ),
  'datadirectory' => '/www/htdocs/server/nextcloud/data',
  'overwrite.cli.url' => 'http://cloud.mydomain.tld',
  'dbtype' => 'mysql',
  'version' => '12.0.2.0',
  'dbname' => 'dbname',
  'dbhost' => 'localhost',
  'dbport' => '',
  'dbtableprefix' => 'xxxxx_',
  'mysql.utf8mb4' => false,
  'dbuser' => 'dbuser',
  'dbpassword' => 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
  'installed' => true,
  'default_language' => 'de',
  'allow_user_to_change_display_name' => false,
  'skeletondirectory' => '',
  'user_backends' => 
  array (
    0 => 
    array (
      'class' => 'OC_User_IMAP',
      'arguments' => 
      array (
        0 => '{mail.mydomain.tld:993/imap/ssl/novalidate-cert/readonly}INBOX',
        1 => 'mydomain.tld',
      ),
    ),
  ),
  'updater.release.channel' => 'production',
  'loglevel' => 2,
  'customclient_desktop' => 'https://nextcloud.com/install/#install-clients',
  'customclient_android' => 'https://f-droid.org/packages/com.nextcloud.client',
  'customclient_ios' => 'https://itunes.apple.com/us/app/nextcloud/id1125420102?mt=8',
  'apps_paths' => 
  array (
    0 => 
    array (
      'path' => '/www/htdocs/server/nextcloud/apps_neu',
      'url' => '/apps_neu',
      'writable' => true,
    ),
    1 => 
    array (
      'path' => '/www/htdocs/server/nextcloud/apps',
      'url' => '/apps',
      'writable' => false,
    ),
  ),
  'memcache.local' => '\\OC\\Memcache\\ArrayCache',
  'theme' => '',
  'filesystem_check_changes' => 1,
  'tempdirectory' => '/www/htdocs/server/nextcloud/tmp/',
  'maintenance' => false,
  'enabledPreviewProviders' => 
  array (
    0 => 'OC\\Preview\\Image',
    1 => 'OC\\Preview\\MP3',
    2 => 'OC\\Preview\\TXT',
    3 => 'OC\\Preview\\MarkDown',
    4 => 'OC\\Preview\\Epub',
    5 => 'OC\\Preview\\FB2',
    6 => 'OC\\Preview\\PDF',
    7 => 'OC\\Preview\\OpenDocument',
    8 => 'OC\\Preview\\StarOffice',
    9 => 'OC\\Preview\\MSOfficeDoc',
    10 => 'OC\\Preview\\MSOffice2003',
    11 => 'OC\\Preview\\MSOffice2007',
  ),
);
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** IMAP

### Client configuration
**Browser:** Firefox

**Operating system:** Linux Mint 17 (based on Ubuntu 14.04)

**Comment by MorrisJobke** [comment]
cc @nextcloud/javascript

**Comment by burned42** [comment]
Hi, I found this issue via the Hacktoberfest and would like to give it a try.

**Comment by burned42** [comment]
As I saw, in e.g. `recentfilelist.js`, `favoritesfilelist,js` the `_setCurrentDir` function is called at `reload()` which sets the page title, so I would call there that in `filelist.js`, too. So I just tested that here and it seems to work just fine.
I also found out that the title also does not get set at the 'Tags' menu entry.

**Comment by danxuliu** [comment]
This seems to have been fixed as a side effect of #6689, and #6987 will fix it too for the system tags.

For reference, there is still a related issue not addressed yet: [the second time (and only the second time) that the _Recent_ or _Deleted files_ sections are opened the page title is not changed](https://github.com/nextcloud/server/issues/6955).
